src/sing/storeHelper.ts

```diff
@@ -4,6 +4,7 @@ export const DEFAULT_TPQN = 480;
 export const DEFAULT_BPM = 120;
 export const DEFAULT_BEATS = 4;
 export const DEFAULT_BEAT_TYPE = 4;
+export const DEFAULT_EDIT_FRAME_RATE = 93.75;
```

Comment:
この値への依存が増えれば増えるほどマルチエンジン化の際に大変になるので、`DEPRECATED_`とつけて非推奨としたいです 🙇 
事情を知らない人がプルリク作るときに気づけるように･･･。

こんな感じにするのはどうでしょう。
```suggestion
export const DEPRECATED_DEFAULT_EDIT_FRAME_RATE = 93.75; // マルチエンジン対応のために将来的に廃止予定で、利用は非推奨
```

Reply:
`DEPRECATED_`をつけてコメントを追加しました！